packages/eslint-plugin/lib/rules/no-untranslated-text.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const {isTextLabelChild, report} = require('../util');
+
+/**
+ * @type {import('eslint').Rule.RuleModule}
+ */
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'enforce text labels in JSX to be wrapped by translate calls',
+      category: 'Suggestions',
+      url: 'https://github.com/facebook/docusaurus/tree/main/packages/eslint-plugin/docs/rules/no-untranslated-text.md',
+    },
+    schema: [],
+    messages: {
+      translateChildren:
+        'All text labels in JSX should be wrapped by translate calls',
+    },
+  },
+
+  create(context) {
+    const isParentTranslate = (child, isParentFragment) =>
+      !isParentFragment &&
+      child.parent.openingElement.name.name === 'Translate';
+
+    const isChildValid = (child, isParentFragment) => {
+      if (!isTextLabelChild({child, includeWhitespace: false})) {
+        return true;
+      }
+      return isParentTranslate(child, isParentFragment);
+    };
```

Comment:
I think this should not report if the text contains only symbols like `·`, `—`, or `×`. I'm not aware of any good heuristics, so maybe we should just add an `ignoreText: ["·", "—", "×"]` option?

Reply:
Does it make sense for each of these symbols to stay the same regardless of the language ? E.g. upside-down question mark in Spanish. And will this case happen frequently enough ? Otherwise, we could also rely on `// eslint-disable-next-line no-untranslated-text` for corner cases like this.

Reply:
You can run `yarn lint:js` locally. Among many other (slightly more justified) cases, the only cases where theme-classic gets reported are those symbols. I think `ignoreText` (or `ignoreCharacters`?) is safe enough—it would only ignore text nodes purely comprised of whitespace and those characters but as soon as there are other characters it would still be reported.

Reply:
Addressed in 7ae0e64. I named it `ignoreStrings` (supports more than one character and `ignoreTexts` sounds a bit weird).